src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSExtendedHearing.java

```diff
@@ -25,13 +26,16 @@ public class CCSExtendedHearing {
     private List<CCSHearingDay> hearingDays;
     private CCSDataSource source;
 
+    private String hearingEventType;
+
     public static CCSExtendedHearing of(Hearing hearing) {
         return CCSExtendedHearing.builder()
                 .caseId(hearing.getCaseId())
                 .hearingId(hearing.getHearingId())
                 .urn(hearing.getUrn())
                 .caseNo(hearing.getCaseNo())
                 .source(CCSDataSource.of(hearing.getSource()))
+                .hearingEventType(hearing.getHearingEventAsType().getDescription())
```

Comment:
This was missing on the first change so CCM never send the event type across to CCS.